chapter12/dcox/ex12_01

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/perl
```

Comment:
Nice!

I only had the operator, not the variable in the value. Don't think that would have made a difference. I think I had it quoted as well and hoped the eval block would work it all out. This is nicer.

Cheers,